datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/db.yaml

```diff
@@ -59,6 +59,8 @@
                  Columns without a name are ignored. To skip a column, enter:
                    - {}
     3. tags (optional) - A list of tags to apply to each metric.
+    4. collection_interval (optional) - The frequency at which to collect the metrics.
+                                       The default is the integration `min_collection_interval`.
```

Comment:
can the collection interval for this ever be lower than the min_collection_interval? I don't think so, right?

Reply:
Right. https://github.com/DataDog/integrations-core/blob/d7b8fff0503c4c39a40ec74ba4a5e348787188be/datadog_checks_base/datadog_checks/base/utils/db/query.py#L42-L44